astroquery/esa/euclid/core.py

```diff
@@ -1276,5 +1277,411 @@ def get_datalinks(self, ids, *, linking_parameter='SOURCE_ID', verbose=False):
+                        SEL-CL to create the random catalogue
+                        #. 'DpdLE3clSelSelFuncInputParams': This product contains the input parameters for SEL SelFunc
+                        #. 'DpdLE3clSelSinfoniaIniClMockInputParams': This product contains the input parameters for
+                        SEL Sinfonia Cluster Mock pipeline
+                        #. 'DpdLE3clSelSinfoniaMockInputParams': This product contains the input parameters for SEL
+                        Sinfonia Mock pipeline
+                        #. 'DpdLE3clSelSelFunc': This output product contains the selection function of the galaxy
+                        cluster detections
+                        #. 'DpdLE3clSinfoniaEllipticity': This output product contains the file for the ellipticity
+                        #. 'DpdLE3clSelSubSample': Clusters Subsample Output Product
+                        #. 'DpdLE3clRedshiftDistrib': Redshift distribution of detections
+                        #. 'DpdLE3clSelRandom': Random for the CL clustering codes
+                        #. 'DpdLE3clMatchForSelParams': This product contains the input parameters for the
+                        match-for-sel pipeline
+                        #. 'DpdLE3clMockClusters': This output product contains the catalog of mock clusters
+                    #. GrpClustering
+                        #. 'DpdLE3clTwoPointAutoCorrPol': Cluster Clustering Two Point Correlation 2D polar data product
+                        #. 'DpdLE3clCovmatTwoPointCov2': Cluster Clustering Covariance Matrix 2PCF 2D polar data product
+                        #. 'DpolDpdLE3clCovmatPKCov1':
+                        #. 'DDpdLE3clPkDOA':
+                        #. 'DpdLE3clPkYam': Cluster Clustering Power Spectrum Yamamoto data product
+                    #. GrpLMF
+                        #. 'DpdLE3clLMFParams': This product contains the configuration parameters for LE3 LMF-CL
+                        #. 'DpdLE3clLMFOutput': This product contains the output for LE3_CL LMF PF
+                    #. GrpTiling
+                        #. 'DpdLE3clCLTile': This product contains the information describing a CL tile
+                        #. 'DpdLE3clCommon': Common data product for CL
+                        #. 'DpdLE3clConfigurationSet': Generic configuration information for CL PFs
+                #. PHZ
+                    #. PHZ
+                        #. 'DpdBinMeanRedshift':
+                        #. 'DpdReferenceSample':
+                        #. 'DpdTomographicBins':
+
+        dataset_release : str, mandatory. Default REGREPROC1_R2
+            Data release from which data should be taken.
+        verbose : bool, optional, default 'False'
+            flag to display information about the process
+
+        Returns
+        -------
+        The list of products (astropy.table)
```

Comment:
I would suggest avoid using 'list' to describe a table as it's slightly misleading
```suggestion
        The products in an astropy.table.Table
```

Reply:
Changes committed to the branch.